spec/mongoid/association/referenced/has_and_belongs_to_many/proxy_spec.rb

```diff
@@ -1757,6 +1757,75 @@
       end
     end
 
+    describe "#any?" do
+
+      let(:person) do
+        Person.create
+      end
+  
+      context "when nothing exists on the relation" do
+        
+        context "when no document is added" do
+
+          let!(:sandwich) do
+            Sandwich.create!
+          end
+
+          it "returns false" do
+            expect(sandwich.meats.any?).to be false
+          end
+        end
+
+        context "when the document is destroyed" do
+  
+          before do
+            Meat.create!
+          end
+  
+          let!(:sandwich) do
+            Sandwich.create!
+          end
+  
+          it "returns false" do
+            sandwich.destroy
+            expect(sandwich.meats.any?).to be false
+          end
+        end
+      end
+
+      context "when appending to a relation and _loaded/_unloaded are empty" do
+  
+        let!(:sandwich) do
+          Sandwich.create!
+        end
+
+        before do
+          sandwich.meats << Meat.new
+        end
+  
+        it "returns true" do
+          expect(sandwich.meats.any?).to be true
+        end
+      end
+
+      context "when appending to a relation in a transaction" do
+        require_topology :replica_set
+        
+        let!(:sandwich) do
+          Sandwich.create!
+        end
+
+        it "returns true" do
+          sandwich.with_session do |session|
+            session.start_transaction
+            expect { sandwich.meats << Meat.new }.to_not raise_error
+            expect(sandwich.meats.any?).to be true
+            session.commit_transaction
+          end
+        end
+      end
+    end
```

Comment:
Could you standardize the tests between habtm/proxy_spec and has_many/proxy_spec? It looks like there are tests in both files that do not exist in the other, which seems inconsistent to me.

Reply:
Agree, which is weird because I just copied #count's tests in each file

Reply:
Fixed this!

Reply:
Should I do the same for embeds_many? or is that one fine